website/source/docs/commands/config.html.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "docs"
+page_title: "Commands: Config"
+sidebar_current: "docs-commands-config"
+---
+
+# Consul Config
+
+Command: `consul config`
+
+The `config` command is used to interact with Consul's central configuration
```

Comment:
For now I think it would be good to link "central configuration system" to the agent option for https://github.com/hashicorp/consul/pull/5734/files#diff-004448710778abf0ba756b61b5e1736aR1113. It will give users a quick overview of centralized config and help them realize it must be enabled. We should do this on the API page too.